benchmark/benchmarkTests.go

```diff
@@ -34,8 +34,10 @@ var (
 	sociBinary        = "../../out/soci-snapshotter-grpc"
 	sociAddress       = "/tmp/soci-snapshotter-grpc/soci-snapshotter-grpc.sock"
 	sociRoot          = "/tmp/lib/soci-snapshotter-grpc"
-	sociConfig        = "../soci_config.toml"
+	snapshotterConfig = "../snapshotter_config.toml"
```

Comment:
how do you distinguish between soci and stargz snapshotter configs?

Reply:
i think you meant [this config](https://github.com/awslabs/soci-snapshotter/blob/main/benchmark/containerd-config.toml). It is used in [getContainerdProcess](https://github.com/awslabs/soci-snapshotter/blob/main/benchmark/benchmarkTests.go#L184) which is used for soci pulling images. So a similar config for stargz will be added in a future PR of adding the functionality of stargz pulling image.

Reply:
I'm still concerned with the former [soci_config.toml](https://github.com/awslabs/soci-snapshotter/blob/main/benchmark/soci_config.toml). With the proposed setup, it will be shared between stargz and soci snapshotters when running different benchmarks. The problem is that these configs are not 1:1 anymore, since they've diverged already. I'm afraid that one of the snapshotters might get misconfigured if we go this way.

Reply:
Separated soci_config and stargz_config in the latest revision.